Dockerfile

```diff
@@ -6,4 +6,5 @@ ADD cmd/out/out /opt/resource/out
 
 RUN chmod +x /opt/resource/*
 
-RUN apk add bash
\ No newline at end of file
+RUN apk add bash
+RUN apk add ca-certificates
```

Comment:
Nit: Does is make sense to use `apk add bash ca-certificates --no-cache` instead. I believe every `RUN` statement gets a separate layer. `--no-cache` allows; https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache